Libraries/WebSocket/RCTWebSocketExecutor.m

```diff
@@ -83,6 +85,8 @@ - (void)setUp
     [self invalidate];
     return;
   }
+  _usageMetricsTimer = [NSTimer scheduledTimerWithTimeInterval:1.0f
```

Comment:
The `RCTWebSocketExecutor` is a bridge module, you can simply add a method to it, and call it from JS to toggle the variable.

As for the frequency of the updates, I don't think it'd be a problem, but I'm ok with leaving as is for now :)